views/index.mustache

```diff
@@ -1,5 +1,4 @@
-{{extends layout}}
+{{> partials/layout}}
 
-{{block content}}
 <h1> {{title}}</h1>
```

Comment:
dont think this alone will work. What about the footer? Partials don't work exactly like block and extend. All the partial will do is take the content from the referred file and just blindly put it there.

Reply:
So, this will render the layout (which has the closing body tag too), then render the h1 tag etc.